packages/dart_frog_cli/lib/src/daemon/domain/dev_server_domain.dart

```diff
@@ -43,6 +43,12 @@ class DevServerDomain extends DomainBase {
       );
     }
 
+    final host = request.params?['host'];
+
```

Comment:
**Nit:** Remove white space to keep consistency with entire file.

```suggestion
```

Reply:
Check [Commit](https://github.com/VeryGoodOpenSource/dart_frog/pull/924/commits/2ba0ff5f665d5c761d846e497377c5184204c536)